src/intents.cs

```diff
@@ -212,8 +212,7 @@ public enum INIntentHandlingStatus : nint {
 		InProgress,
 		Success,
 		Failure,
-		DeferredToApplication,
-		Done = Success
+		DeferredToApplication
```

Comment:
minor: in the future add `,` on the last enum items. It's legal (in C#) and makes smaller diffs when adding members.